tests/console/nvme_checks.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Smoke tests for NVM Express Controller.
+#
+# - Install nvme-cli which is a tool to manage NVM Express
+# - Check sys path that should be hold details
+# - Check if we have any NVMe controllers on the PCI bus
+# - Check if we have NVMe devices
+# - Check if we have namespace
+# - Send Identify Admin Command to the NVMe controller
+# - Issue a Read command both using the helper and manually
+# - Compare it to what we have with a regular dd
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use scheduler 'get_test_suite_data';
+
+sub _smoke_checks {
+    my $nvm_test_data = shift;
+    assert_script_run("lspci -nn | grep -i nvm");
```

Comment:
I am not expert on this but this is the think. from the output [here](http://aquarius.suse.cz/tests/3057#step/nvme_checks/13) you can see that it says `QEMU NVM Express` blabla. But the output can be different. For instance i found that it can show `NVMe` instead of `NVM Express`. So i went for the most general. However yes, it is always Express.

For the second part of the comment, this line is unlikely to change and i think is overkill to pass it as test_data. I am intending to provide a comment if you think it would make it easier to understand. is it ok?

Reply:
I found the same when reading about it, NVMe is also correct, I'd rather choose some way where express were used.
Probably is a constant for the test, that is right, no need for test data.